pages/index.js

```diff
@@ -1,34 +1,35 @@
-import Link from "next/link";
 import styled from "styled-components";
-import FormToCreateShoppingItem from "@/components/FormToCreateShoppingItem.js";
+import FormToCreateShoppingItem from "@/components/FormToCreateShoppingItem/FormToCreateShoppingItem.js";
+import ShoppingItem from "@/components/ShoppingItem/ShoppingItem.js";
+
 import { categories } from "@/lib/categoriesData";
 
 export default function ShoppingItemsList({
```

Comment:
Consider using simpler Code for the ShoppingItemsList - component:

export default function ShoppingItemsList({
    shoppingItemsWithCategoryColor,
    onAddItem,
    onDeleteItem,
  }) {
    return (
      <main>
        <StyledH1>{shoppingItemsWithCategoryColor.length} Shopping Items</StyledH1>
        {shoppingItemsWithCategoryColor.length === 0 && (
          <StyledEmptyMessage>
            There are no items on your shopping list. Add items using the form
            below.
          </StyledEmptyMessage>
        )}
  
        <FormToCreateShoppingItem onAddItem={onAddItem} categories={categories} />
  
        <StyledList>
          {shoppingItemsWithCategoryColor.map((shoppingItem) => (
            <ShoppingItem
              key={shoppingItem.id}
              shoppingItem={shoppingItem}
              onDeleteItem={onDeleteItem}
            />
          ))}
        </StyledList>
      </main>
    );
  }

Reply:
Hey Johann, thanks a lot for your comments! :)
I have taken out the fragments. 
Regarding your other comments, I have decided to stick with the styled message frame for now because it gives me the opportunity to display the message in a box, with a coloured background, etc.
Regarding the second suggested simplification - if I take out the return, the items are not displayed below the form. Therefore, I have left it as it is, too. 
I will probably have to work on the layout in order to pass the quality assurance...